### DISCUSSION TITLE
"Always Allow" Instant "interaction-policy" doesn't work

### DISCUSSION THREAD

**Discussion by ferb7nft** [google/model-viewer#2589] [Q&A] [answered]
I tried adding 'interaction-policy' (even though it should be 'always-allow' by default). I can instantly react with the 3D model on desktop but still need to "focus"/tap on the model on mobile phones before I can interact. How can I make it so I can instantly interact with it on mobile as well without needing to tap the viewer first? (the problem is on iOS, I'm not sure about Android). Thank you :) 

<model-viewer
  src="file.glb"
  ios-src="file.usdz"
  ar-modes="webxr scene-viewer quick-look"
  camera-controls
  loading="eager"
  auto-rotate
  auto-rotate-delay="0"
  shadow-intensity="0"
  ar
  interaction-policy="always-allow"
  interaction-prompt-threshold="0"
  interaction-prompt-style="wiggle"
  >
</model-viewer>

**Comment by elalish** [comment]
That's not the behavior I'm seeing with iOS on our example pages. Can you post some detailed repro steps, including the page you're testing?

**Reply by ferb7nft** [reply]
Yes I will add the website shortly, thank you!

**Reply by ferb7nft** [reply]
Here is the website:

https://ipfs.io/ipfs/QmPLG92kQuXjGVuv74gbuXpfoxiBAuzmmfAaNGe9cBcZt4/

**Reply by ferb7nft** [reply]
Here is the code: 
[index.txt](https://github.com/google/model-viewer/files/6842480/index.txt)

**Reply by elalish** [reply]
Oh, well that's your problem! You're using an ancient version of model-viewer! It wasn't even API stable yet. I'd recommend removing the version string to get the latest automatically, or explicitly use our most recent: 1.7.2. This is likely the source of your other issues as well. Also, we no longer support IE, so you don't need the legacy version anymore.

**Reply by ferb7nft** [reply]
And this website and code also does not work

**Reply by ferb7nft** [reply]
Oh thank you!! I didn't know that, I just followed the documentation on the model-viewer website! Maybe that should be updated @elalish

**Reply by ferb7nft** [reply]
I will test now and reply @elalish also, what would the api call look like if I "removed the version string"?

**Reply by elalish** [reply]
Where did you copy that line from? I thought I had updated that everywhere it existed, but I must have missed one. It'll look like: `<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>` just like the front page of modelviewer.dev.

**Reply by ferb7nft** [reply]
I think you are correct, it now looks updated on the model viewer website. Did you just update it within a week? Cause I grabbed the code before that. Otherwise, one day I might have accidentally updated it with old code from a stack overflow example haha. Anyways, it is updating now, and I will let you know if it works!

**Reply by ferb7nft** [reply]
Wow it worked! Thank you so much! There is strange shading on it now, but I will figure that out as another problem. But the new version solved the original problem, thank you!

**Comment by ferb7nft** [comment, marked as answer]
The problem was an outdated api version. It was fixed by using <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
